jumeg_causality/dirs_manage.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
@pravsripad, Yes I sent this information directly from Evernote, I thought it worked, we can delete them. @jdammers, can you also register one evernote account? I am checking the ICs related with ECG&EOG artifacts, and finishing reports of the preprocessing step on the evernote. You can check the report under the sharing notebook made by @pravsripad.